### PR TITLE
Fix CaseClauseError in fetch_user for {:error, Response} tuple

### PR DESCRIPTION
**Pull request by davydog187**

### CODE REVIEW

**Comment by davydog187** [comment]
Addresses #39 partially

File: lib/ueberauth/strategy/google.ex [hunk 1]
```diff
@@ -128,6 +128,8 @@ defmodule Ueberauth.Strategy.Google do
         set_errors!(conn, [error("token", "unauthorized")])
       {:ok, %OAuth2.Response{status_code: status_code, body: user}} when status_code in 200..399 ->
         put_private(conn, :google_user, user)
+      {:error, %OAuth2.Response{status_code: status_code}} ->
+        set_errors!(conn, [error("OAuth2", status_code)])
```
Comment (doomspork) [inline lib/ueberauth/strategy/google.ex]: Should we perhaps grab the body and use that as the error reason in place of the status code?

Comment (davydog187) [reply lib/ueberauth/strategy/google.ex]: In my case, the body was an empty map, so probably not

Comment (davydog187) [reply lib/ueberauth/strategy/google.ex]: See my comment here https://github.com/ueberauth/ueberauth_google/issues/39#issuecomment-345282172

Comment (doomspork) [reply lib/ueberauth/strategy/google.ex]: Okay.  Let's go with this for now, someone can always open a PR later 👍

**Comment by davydog187** [comment]
🍍 🍍 🍍 🍍 🍍 🍍 🍍 🍍

**Comment by doomspork** [comment]
I cut a new release @davydog187 👍

**Comment by davydog187** [comment]
thank you @doomspork ! You are very fast 🚗

**Pull request merged**